src/main.rs

```diff
@@ -108,6 +110,14 @@ fn main() {
             }
         }
 
+        if opt.summary {
+            match result.exit_status {
+                ExitStatus::Exited(0)  =>  summary.successes += 1,
+                ExitStatus::Exited(n) => summary.failures.push(n),
+                _ => summary.failures.push(99),
```

Comment:
Can you document what this 99 value is?

Reply:
I needed  some i32 for "any other failure", I'll add a comment.